watertap/costing/tests/test_costing_base.py

```diff
@@ -134,3 +135,40 @@ def test_watertap_costing_package():
     m.fs.costing.capital_recovery_factor.fix()
     # no error, wacc, capital_recovery_factor fixed
     m.fs.costing.initialize()
+
+
+@pytest.mark.component
+def test_LCOW_breakdowns():
```

Comment:
Since units are so critical to these breakdowns, I am wondering if a ```assert_units_consistent``` would be valuable somewhere in here?

Reply:
Done in 6f6d1a7